components/classroom/test.html

```diff
@@ -8,6 +8,20 @@
 <body>
     <h1>Classroom Group Management</h1>
     <p id="project-info">Project ID: </p>
-    <script src="project.js"></script>
+        <script type="module">
+        import { fetchProjectData } from './fetchProjectData.mjs';
+        import { getProjectIDFromURL } from './project.js';
+
+        // Call the functions
+        const projectId = getProjectIDFromURL();
+        if (projectId) {
+            // Fetch and display project data
+            fetchProjectData(projectId).then(data => {
+                document.getElementById('project-info').textContent = `Project Info: ${JSON.stringify(data)}`;
+            });
+        } else {
+            document.getElementById('project-info').textContent = 'Project ID not found in URL';
```

Comment:
When I run the test program, I don't get an error, but no projectID is printed. Since there aren't any elements for our program to actually get, we would expect the error message to print, right? This might be a non-problem once we actually have working project ids in the interface.
![image](https://github.com/user-attachments/assets/3cf1232e-c4fc-48da-bfc7-d007bb2587e7)